flow/payload.yaml

```diff
@@ -0,0 +1,47 @@
+components:
+  schemas:
+    Flow.Payload:
+      description: |-
+        A container for different types of payload, which is 
+        the data in the frame after protocol headers and instrumentation bytes. 
```

Comment:
Need to correct the description.
Payload data will start immediately after protocol header, not after instrumentation bytes. Some part of the payload will be overwritten with instrumentation data. Contents of instrumentation data and placement is implementation specific.

Reply:
updated, Thanks